pkg/internal/netpolerrors/netpol_errors.go

```diff
@@ -126,3 +141,25 @@ const colonSep = ": "
 func ConcatErrors(err1, err2 string) string {
 	return err1 + colonSep + err2
 }
+
+// SamePriorityErr returns the error message if a priority appears more than once in different admin-network-policies
+func SamePriorityErr(name1, name2 string) string {
+	return "Admin Network Policies: " + name1 + " and " + name2 + " have same priority;" +
+		"Two policies are considered to be conflicting if they are assigned the same priority."
+}
+
+// PriorityValueErr returns error message of invalid priority value in an admin-network-policy
+func PriorityValueErr(name string, priority int32) string {
+	return fmt.Sprintf("Invalid Priority Value: %d in Admin Network Policy: %q; Priority value must be between 0-1000", priority, name)
```

Comment:
use const values for valid priority values instead of hard-coded ones

Reply:
done

Reply:
why isn't pkg netpolerrors under `pkg/netpol/internal` ?
I think it may be better to define these consts under pkg named `common` and not under pkg `netpolerrors`

Reply:
`netpolerrors` is used also in `cli` and `manifests` (can not be moved to `netpol\internal`)
i agree `netpolerrors` it is not the best pkg to put those constants. (but `netpolerrors` is the only seen to both now, don't think worth to add new pkg because of two contants.)

Reply:
done, new issue #446